site/content/docs/contributing/developer-docs/repo-structure.md

```diff
@@ -19,10 +19,12 @@ Some of these directories include:
 - `sdk/` - Contains the plugin [software development kits (SDKs)][plugin_sdk] maintained by the Hipcheck
 	team for various languages, which each language's SDK in a separate subdirectory.
 	- `rust/` - The Hipcheck [Rust SDK][rust_sdk] crate.
-- `hipcheck-common/` - An internal library crate containing functionality shared between the `hc` binary crate and the Rust SDK.
-	- `proto/` - The Protobuf protocol definition for communication between the `hc` binary and plugins.
-- `hipcheck-macros/` - An internal library crate of proc macros for the `hc` binary.
-- `hipcheck-sdk-macros/` - An internal library crate of proc macros for the Rust SDK.
+- `library/` - Supporting library crates used by Hipcheck core or by plugins.
+  - `hipcheck-common/` - An internal library crate containing functionality shared between the `hc` binary crate and the Rust SDK.
+  - `hipcheck-kdl/` - An internal library crate for working with KDL files.
+  - `hipcheck-macros/` - An internal library crate of proc macros for the `hc` binary.
+  - `hipcheck-sdk-macros/` - An internal library crate of proc macros for the Rust SDK.
+- `proto/` - The Protobuf protocol definition for communication between the `hc` binary and plugins.
```

Comment:
It looks like `proto/` is still under `hipcheck-common/` in the changes. Is it meant to be at the top level instead, or is the docs change wrong?

Reply:
Ah, there's a copy of it under `proto/` and a copy at the root, which I thought we'd resolved previously to have one be a symlink of the other.

At this point I am inclined to make the one under `common/` canonical and remove it from the root entirely if that doesn't actually break anything.